packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/config_with_multiple_when.yaml

```diff
@@ -0,0 +1,48 @@
+# config/packages/webpack_encore.yaml
+webpack_encore:
+    # ...
+    output_path: '%kernel.project_dir%/public/build'
+    strict_mode: true
+    cache: false
+
+# cache is enabled only in the "prod" environment
+when@prod:
+    webpack_encore:
+        cache: true
+
+# disable strict mode only in the "test" environment
+when@test:
+    webpack_encore:
+        strict_mode: false
+
+# YAML syntax allows to reuse contents using "anchors" (&some_name) and "aliases" (*some_name).
+# In this example, 'test' configuration uses the exact same configuration as in 'prod'
+when@prod: &webpack_prod
+    webpack_encore:
```

Comment:
@tacman @TomasVotruba it seems Yaml Parser mark it as duplicated key:

```bash
There was 1 error:

1) Symplify\ConfigTransformer\Tests\Converter\ConfigFormatConverter\YamlToPhp\YamlToPhpTest::testNormal with data set #1 (Symplify\SmartFileSystem\SmartFileInfo Object (...))
Symfony\Component\Yaml\Exception\ParseException: Duplicate key "when@prod" detected at line 20 (near "    # ...").

/Users/samsonasik/www/symplify/vendor/symfony/yaml/Parser.php:336
```

Reply:
I removed duplicated key https://github.com/symplify/symplify/pull/4179/commits/c693013113942968444a6de669396e7964794f30 so we can focus on converting it first.